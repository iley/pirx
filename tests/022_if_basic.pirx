extern func main(): int {
  var x: int = 1;
  var y: int = 10;

  if x < 10 {
    printf("x is less than 10\n");
  }

  if y < 10 {
    printf("y is less than 10\n");
  }

  if x < y {
    printf("x is less than y\n");
  } else {
    printf("x is not less than y\n");
  }

  if x > y {
    printf("x is greater than y\n");
  } else {
    printf("x is not greater than y\n");
  }

  return 0;
}
