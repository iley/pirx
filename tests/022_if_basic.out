x is less than 10
x is less than y
x is not greater than y
